Wrapping/Python/PythonDocstrings.i

```diff
@@ -4730,7 +4730,12 @@ passes as the initial translation to the transform. This second
 approach assumes that the moments of the anatomical objects are
 similar for both images and hence the best initial guess for
 registration is to superimpose both mass centers. Note that this
-assumption will probably not hold in multi-modality registration.   \\\\sa itk::CenteredTransformInitializer
+assumption will probably not hold in multi-modality registration.
```

Comment:
Same comment as in the java wrapping:
...assumption **is less likely** to hold in multi-modality registration.
the proposed text is slightly less conclusive as the original (for instance CT/MR registration of the head ...)